ui/package-lock.json

```diff
@@ -38210,6 +38220,11 @@
             "slash": "^3.0.0"
           }
         },
+        "immer": {
+          "version": "8.0.1",
```

Comment:
8.0.1 has CVE, so as long as we are keeping it in the repo, we are vulnerable right ?

Reply:
TBH, I do not know how it works. This is a nested dep. from `react-dev-utils`. The top-level entry for `immer` is at `9.0.6` now.

Reply:
Is there a way to verify what is actually packaged?

Reply:
I too have no idea for this side of the code. Theoretically the current changes looks ok to me. But better to consult UI experts :)

Reply:
It turned out to be complicated... I did another force push to use `npm-force-resolutions` for bumping nested `immer` dependencies to `9.0.6`.

Reply:
It shows 34KLOC diff, do we really need to handle this way ? I think we can consult `react-dev-utils` and ask them for a formal release ?